bigflow/_version.py

```diff
@@ -1,2 +1,2 @@
-__version__ = '1.4.2rc2'
+__version__ = '1.4.2rc3'
```

Comment:
shouldn't be just 1.4.2 if you are going to release a version 1.4.2?

Reply:
I will release the final rc5, deploy it to the production in our system, then release 1.4.2